mirage/views/utils/index.ts

```diff
@@ -43,6 +43,9 @@ export function filter(model: ModelInstance, request: Request) {
                     if (field === 'id') {
                         return compareIds(model.id, val, toOperator(operator));
                     }
+                    if (model[`${field}Ids`] === null || model[`${field}Id`] === null) {
```

Comment:
So if I want to filter a non-relationship field, would `model[`${field}Ids`]` be null?

Reply:
Actually, if you look at this `filter` method, it's trying to manage three different cases: 1. filtering by object's id. 2. filtering by relationship's id. 3. filtering by model attribute. The case we're trying to handle now falls into the second category, just that the handling of the `actualValue` is a bit different. So instead of adding a new `if` clause here, we might be able to modify the `compareIds` hook directly.

Reply:
The case we're trying to handle doesn't get caught in the second category though, since the `null` relationshipId is falsey. We could update the [conditional to check relationshipId](https://github.com/CenterForOpenScience/ember-osf-web/blob/develop/mirage/views/utils/index.ts#L46) to check if `relationshipId` or `relationshipIds` is `null`, and adjust the first argument of `compareIds` to account for that? something like:

```
if (model[`${field}Ids`] || model[`${field}Id`] || (model[`${field}Ids`] === null || model[`${field}Id`] === null) ) {
    return compareIds(model[`${field}Ids`] || model[`${field}Id`] || '', val, toOperator(operator));
}

Reply:
That sounds like a better solution.